docs/source/en/index.mdx

```diff
@@ -50,7 +50,6 @@ The library currently contains JAX, PyTorch and TensorFlow implementations, pret
 ### Supported models
 
 <!--This list is updated automatically from the README with _make fix-copies_. Do not update manually! -->
-
```

Comment:
Not sure what happened here, but normally there should only be 2 added lines in index.mdx (namely, the new model)